src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -1,33 +1,81 @@
-import React from 'react';
-import { reduxForm } from 'redux-form';
+import _ from 'lodash';
+import React, { Component } from 'react';
+import { reduxForm, initialize } from 'redux-form';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
 import validate from './validate';
+import TextField from '../form-elements/TextField';
+import SelectField from '../form-elements/SelectField';
+import ArrayField from '../form-elements/ArrayField';
+import ButtonField from '../form-elements/ButtonField';
+import { renderFormField } from '../../utils/form-utils';
+import { PRODUCTS_MOCKS } from '../../mockedData';
 
-const AddItemsPage = (props) => {
-  const { handleSubmit, previousPage } = props;
-
-  return (
-    <form onSubmit={handleSubmit}>
-      ADD ITEMS PAGE
-      <div>
-        <button type="button" className="btn btn-outline-primary" onClick={previousPage}>
-          Previous
-        </button>
-        <button type="submit" className="btn btn-outline-primary">Next</button>
-      </div>
-    </form>
-  );
+const FIELDS = {
+  lineItems: {
+    type: ArrayField,
+    addButton: 'Add line',
+    fields: {
+      productCode: {
+        type: SelectField,
+        label: 'Requisition items',
+        attributes: {
+          openOnClick: false,
+          options: PRODUCTS_MOCKS,
+        },
+      },
+      quantity: {
+        type: TextField,
+        label: 'Quantity',
+      },
+      deleteButton: {
+        type: ButtonField,
+        label: 'Delete',
+        buttonLabel: 'Delete',
+        getDynamicAttr: ({ removeRow }) => ({
```

Comment:
Can you explain how this works?

Reply:
when you want to remove a row, it's need to be removed from the ArrayField which is a parent of this button, so to accomplish this ArrayField pass a callback function (in the attributes) to each of its children (different function for every row)